src/pages/Cart/CartInfoBox/CartInfoBox.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import InfoMapBox from './InfoMapBox';
+import './CartInfoBox.scss';
+
+export default function CartInfoBox({ infoBox }) {
+  const [sum, setSum] = useState(0);
```

Comment:
```suggestion
  const [price, setPrice] = useState(0);
```
저는 sum 보다는 price가 좀더 직관적으로 보이는데 어떠실까요?

Reply:
그럼 총합 금액을 구하는 것이기 때문에, 더 좋은 이름을 생각해봐야겠네요 감사함니다!